modules/navier_stokes/src/linearfvkernels/LinearFVEnergyAdvection.C

```diff
@@ -0,0 +1,107 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "LinearFVEnergyAdvection.h"
+#include "MooseLinearVariableFV.h"
+#include "NSFVUtils.h"
+#include "NS.h"
+
+registerMooseObject("MooseApp", LinearFVEnergyAdvection);
+
+InputParameters
+LinearFVEnergyAdvection::validParams()
+{
+  InputParameters params = LinearFVFluxKernel::validParams();
+  params.addClassDescription("Represents the matrix and right hand side contributions of an "
+                             "advection term for the energy e.g. h=cp*T. A user may still "
+                             "override what quantity is advected, but the default is cp*T.");
```

Comment:
```suggestion
                             "override what quantity is advected, but the default is the specific enthalpy");
```
what specific enthalpy means is defined elsewhere than the kernel